### PR TITLE
docs(#211): add more details to API document of v1

### LINKED ISSUE #211: The API document of v1 needs to be more detailed.
**What you expected to happen**

- The API document of v1 needs to be more detailed and more human readable.

### PR DESCRIPTION
**Pull request by zz-bmj**
## Fixes #211

### CODE REVIEW

**Comment by zz-bmj** [comment]
doc about WebSocket and error response is needed.

**Comment by zz-bmj** [comment]
change base branch to v1-preview and cherry pick API doc.
PTAL @supereagle

File: docs/api/v1/api.md [hunk 1]
```diff
@@ -0,0 +1,821 @@
...
+{
+    "status": ""  // string, required
+}
+```
+
+Note:
+
+| Field | Note |
+| --- | --- |
+| status | Aborted only |
```
Comment (supereagle) [inline docs/api/v1/api.md]: Only support `Aborted` status.

File: docs/api/v1/api.md [hunk 2]
```diff
@@ -0,0 +1,821 @@
...
+### ProjectDataStructure
+
+```
+{
+  "creationTime": "2017-08-23T09:44:08.653Z",   // created time of the project
+  "description": "string",                      // description of the project
+  "lastUpdateTime": "2017-08-23T09:44:08.653Z", // updated time of the project
+  "name": "string",                             // name of the project, should be unique
+  "owner": "string",                            // owner of the project
+  "tenant":"string"
```
Comment (supereagle) [inline docs/api/v1/api.md]: Not needed

File: docs/api/v1/api.md (hunk 1 above)
Comment (zz-bmj) [reply docs/api/v1/api.md]: done

File: docs/api/v1/api.md (hunk 2 above)
Comment (zz-bmj) [reply docs/api/v1/api.md]: ok

**Pull request merged**